src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -31,14 +32,19 @@
   host_pattern: hostname
 data_devices:
   paths:
-  - /dev/sda"""
+  - data: /dev/sda
+    crush_device_class: ssd"""
     ),
 ])
 def test_DriveGroup(test_input):
+
     dg = DriveGroupSpec.from_json(yaml.safe_load(test_input))
     assert dg.service_id == 'testing_drivegroup'
     assert all([isinstance(x, Device) for x in dg.data_devices.paths])
-    assert dg.data_devices.paths[0].path == '/dev/sda'
+    if isinstance(dg.data_devices.paths[0].path, str):
```

Comment:
was mypy complaining without this extra `isinstance`? It seems like the `path` attribute for the `Device` class is still just type `str` so it feels like this shouldn't be necessary.

Reply:
Yes, this change just solved that problem. I'll add a stacktrace to show you the issue, but definitely having this check (that doesn't change the nature of the test) helped to make it happy